widgets/BigQueryTable-batchsink.json

```diff
@@ -48,22 +48,59 @@
           }
         },
         {
-          "widget-type": "radio-group",
+          "name": "createPartitionedTable",
+          "widget-type": "toggle",
+          "label": "Create Partitioned Table",
+          "widget-attributes": {
+            "on": {
+              "value": "true",
+              "label": "On"
```

Comment:
It sounds a bit weird to say 'Create Partitioned Table' is 'On'. I believe other places just use capitalized 'True' and 'False' as the label.

Reply:
Labels was changed.